lib/internal/test_runner/harness.js

```diff
@@ -31,12 +37,16 @@ function createProcessEventHandler(eventName, rootTest, testResources) {
 
       test.fail(new ERR_TEST_FAILURE(err, eventName));
       test.postRun();
+    } else {
+      console.error(err);
```

Comment:
There's no need to require `console`. I do wonder if this is how we want to handle this case, however. We will rarely just blindly log things out to console like this.

Reply:
I was thinking maybe throw error in this case